rts/motoko-rts-tests/src/principal_id.rs

```diff
@@ -36,11 +36,11 @@ pub unsafe fn test() {
 
     let text = text_of_str(&mut heap, "aaaaa-aa");
     let principal = blob_of_principal(&mut heap, text);
-    assert_eq!(text_compare(principal, text_of_str(&mut heap, ""),), 0);
+    assert_eq!(blob_compare(principal, text_of_str(&mut heap, ""),), 0);
 
     let text = text_of_str(&mut heap, "bfozs-kwa73-7nadi");
     assert_eq!(
-        text_compare(
+        blob_compare(
```

Comment:
If I am not mistaken, this compares a `TAG_BLOB_B` with `TAG_BLOB_T`. Perhaps, the `text_of_str()` is here more like a blob encoder in this specific test...

Reply:
Yes, it is. I changed to blob_compare because text_compare (that deals with ropes) checks the TAG_BLOB_T tags of leaves. bob_compare only checks the contents.